css/Sass/profile.scss

```diff
@@ -0,0 +1,22 @@
+
+.profile {
+	h1 {
+		font-size: 48px;
+		position: initial;
+			small {
```

Comment:
Perhaps not.  The only thing that is different is the position (absolute in main; here it is initial).  There is probably a better way of handling both situations.